packages/Webkul/Velocity/src/Resources/assets/js/UI/components/product-card.vue

```diff
@@ -109,6 +111,10 @@
                 } else {
                     return false;
                 }
+            },
+
+            'redirectToReview': function() {
+                window.location.href = `product/${this.product.slug}/review`
```

Comment:
add base url also,

this.$root.baseUrl

Reply:
https://github.com/bagisto/bagisto/pull/7316/commits/cd9d4ff7c8cd1610913b06db9b43685d38cb8553

Sir Already done using simple anchor tag

Reply:
Then remove this method if not needed.

Reply:
I removed that method.